packages/core/package.json

```diff
@@ -8,26 +8,38 @@
   "scripts": {
     "lint": "eslint \"**/*.ts*\"",
     "build": "rm -rf dist && tsup index.ts",
+    "test": "jest",
     "prepare": "cp ../../README.md . && yarn build",
     "postpublish": "rm README.md"
   },
   "files": [
     "dist"
   ],
   "devDependencies": {
+    "@testing-library/react": "^14.0.0",
+    "@types/deep-diff": "^1.0.3",
+    "@types/event-emitter": "^0.3.3",
+    "@types/jest": "^29.5.4",
     "@types/react": "^18.2.0",
     "@types/react-beautiful-dnd": "^13.1.4",
     "@types/react-dom": "^18.2.0",
     "eslint": "^7.32.0",
     "eslint-config-custom": "*",
     "react-feather": "^2.0.10",
+    "jest": "^29.6.4",
+    "jest-environment-jsdom": "^29.6.4",
+    "ts-jest": "^29.1.1",
     "tsconfig": "*",
     "tsup": "^6.7.0",
     "tsup-config": "*",
     "typescript": "^4.5.2"
   },
   "dependencies": {
+    "deep-diff": "^1.0.2",
+    "event-emitter": "^0.3.5",
     "react-beautiful-dnd": "^13.1.1",
+    "react-feather": "^2.0.10",
```

Comment:
This looks like a merge conflict - react-feather should have been removed in favour of the `devDependency` (see #121)